File: Foundation/Arrays/frequency.cpp
// #include<iostream>
// #include<vector>
// using namespace std;

// void freq(vector<int>& arr)
// {
//     int num;
//     cin >> num;
//     int count = 0;
//     for(int i = 0; i < arr.size(); i++)
//     {
//         if(arr[i] == num)
//         {
//             count++;
//         }
//     }
//     cout<<count;
// }

// int main()
// {
//     int n;
//     cin >> n;
//     vector<int> arr(n);
//     for(int i = 0; i < arr.size(); i++)
//     {
//         cin>>arr[i];
//     }
//     freq(arr);
//     return 0;
// }
#include<iostream>
using namespace std;

int freq(int num, int data)
{
    int count = 0;
    int mod;
    while(num > 0)
    {
        mod = num % 10;
        if(mod == data)
        {
            count++;
        }
        num /= 10;
    }
    return count;
}

int main()
{
    int num, data;
    cin>>num;
    cin>>data;
    cout<<freq(num, data);
    return 0;
}

File: Foundation/Arrays/frequency.java
// import java.util.Scanner;
// public class frequency
// {
//     public static Scanner scn = new Scanner(System.in);
//     public static void freq(int[] arr)
//     {
//         int num = scn.nextInt();
//         int count = 0;
//         for(int i = 0; i < arr.length; i++)
//         {
//             if(arr[i] == num)
//             {
//                 count++;
//             }
//         }
//         System.out.println(count);
//     }

//     public static void main(String[] args)
//     {
//         int n = scn.nextInt();
//         int[] arr = new int[n];
//         for(int i = 0; i < arr.length; i++)
//         {
//             arr[i] = scn.nextInt();
//         }
//         freq(arr);
//     }
// }
import java.util.Scanner;
public class frequency
{
    public static Scanner scn = new Scanner(System.in);
    public static int freq(int num, int data)
    {
        int count = 0;
        int mod;
        while(num > 0)
        {
            mod = num % 10;
            if(mod == data)
            {
                count++;
            }
            num /= 10;
        }
        return count;
    }

    public static void main(String[] args)
    {
        int num = scn.nextInt();
        int data = scn.nextInt();
        System.out.println(freq(num, data));
    }
}